src/components/OvalBanner.tsx

```diff
@@ -36,16 +66,14 @@ export default function OvalBanner({ isLightTheme: isLightTheme_ = false }) {
             <Icon name="info" className="h-4 w-4 text-red" />
           </div>
           <div className="text-[--color]">
-            <span className="hidden sm:inline">
-              Introducing UMA Oval: An MEV capture tool that allows you to monetize liquidations in your protocol.
-            </span>
+            <span className="hidden sm:inline">{copy}</span>
             <span className="sm:hidden">Introducing UMA Oval</span>
           </div>
         </div>
         <div>
           <NextLink
             className="flex items-center gap-1 text-[--color] transition hover:opacity-50"
```

Comment:
can you tilt the arrow 45 degrees to indicate its goign outside the site

Reply:
okay 👍